evalml/tests/component_tests/test_ft_transform_primitive_components.py

```diff
@@ -225,10 +225,23 @@ def test_component_fit_transform(
     component.fit(data)
     new_X = component.transform(data)
 
-    pd.testing.assert_frame_equal(new_X, expected)
+    # TODO: Understand why the "EMAIL_ADDRESS_TO_DOMAIN(email)" engineered feature
```

Comment:
I think the new change in behavior with pandas 1.4.0 is that the `categories` attribute of the categorical dtype now includes "original" feature values. Since the physical type of `URL`/`EmailAddress` logical types is "string" (nullable), that's why the categories  have nullable values.

I think we can get around this behavior if we update the `make_answer` functions but I think what we have here is fine too.

Reply:
Do you know how to do that?  I pinged the woodwork team for how we can add in that specific column with the "string" type, but I can't seem to do it.

Reply:
I think like this:

![image](https://user-images.githubusercontent.com/41651716/154337849-98ef7765-152c-471c-bd68-0a644be9d145.png)

Reply:
Not blocking though. The current equality logic in the test is sufficient.